website_docs/getting-started.md

```diff
@@ -54,6 +54,8 @@ with tracer.start_as_current_span("foo"):
     with tracer.start_as_current_span("bar"):
         with tracer.start_as_current_span("baz"):
             print("Hello world from OpenTelemetry Python!")
+
```

Comment:
can we add a comment describing why we are doing this here so people don't blindly copy it?

Reply:
Added, hope it makes sense.